components/Button/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import {StyleSheet, View, TouchableOpacity, Text} from 'react-native'
+import PropTypes from 'prop-types'
+
+export default class Button extends Component {
```

Comment:
If you ever find yourself writing a class with the only method being `render()`, it's probably better off as a stateless functional component